scalafix-core/src/main/scala/scalafix/internal/patch/EscapeHatch.scala

```diff
@@ -23,13 +24,13 @@ import scalafix.v0._
  * precedence over the former in case there are overlaps.
  * See `AnchoredEscapes` and `AnnotatedEscapes` for more details.
  */
-class EscapeHatch private (
+private[scalafix] class EscapeHatch private (
```

Comment:
Note: it's generally not necessary to mark things private in `scalafix.internal` unless it helps readability with highlighting the key components from implementation details

Reply:
Reverted